src/context/example.rs

```diff
@@ -0,0 +1,53 @@
+use report::example::ExampleReport;
+
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub enum ExampleLabel {
```

Comment:
not a big fan of this struct.

Thought: use [strum](https://github.com/Peternator7/strum) which does the same via a macro ? Won't make it away, but it's less code